site/src/components/navbar.tsx

```diff
@@ -180,6 +187,14 @@ export const Navbar: FunctionComponent<NavbarProps> = ({ openLoginModal }) => {
     document.body.style.overflow = mobileNavVisible ? "hidden" : "auto";
   }, [mobileNavVisible]);
 
+  const handleLoginButtonClick = useCallback<MouseEventHandler>(
```

Comment:
Should prevent [Cancel rendering route](https://sentry.io/organizations/hashintel/issues/3500395414/?project=6603021& query=is%3Aunresolved) error (happens when we click Log In and url switches to `#`).